src/pages/workspaces/workspace/Dashboard.js

```diff
@@ -185,6 +186,46 @@ const BucketLocation = requesterPaysWrapper({ onDismiss: _.noop })(({ workspace
   return h(TooltipCell, [flag, ' ', regionDescription])
 })
 
+export const WorkspaceNotifications = ({ workspace }) => {
+  const { workspace: { namespace, name } } = workspace
+
+  const [saving, setSaving] = useState(false)
+
+  const notificationsPreferences = _.pickBy((_v, k) => _.startsWith('notifications/', k), authStore.get().profile)
+
+  const submissionNotificationKeys = [
+    `notifications/SuccessfulSubmissionNotification/${namespace}/${name}`,
+    `notifications/FailedSubmissionNotification/${namespace}/${name}`,
+    `notifications/AbortedSubmissionNotification/${namespace}/${name}`
+  ]
+
+  const submissionNotificationsEnabled = !_.isMatch(
+    _.fromPairs(_.map(k => [k, 'false'], submissionNotificationKeys)),
+    notificationsPreferences
+  )
+
+  return div({ style: { margin: '0.5rem' } }, [
+    div({ style: { display: 'flex', alignItems: 'center' } }, [
+      h(LabeledCheckbox, {
+        checked: submissionNotificationsEnabled,
+        disabled: saving,
+        onChange: _.flow(
+          Utils.withBusyState(setSaving),
+          withErrorReporting('Error saving preferences')
+        )(async value => {
+          await Ajax().User.profile.setPreferences(_.fromPairs(_.map(k => [k, JSON.stringify(value)], submissionNotificationKeys)))
+          await refreshTerraProfile()
+          Ajax().Metrics.captureEvent(Events.notificationToggle, { notificationKeys: submissionNotificationKeys, enabled: value })
+        })
+      }, [span({ style: { marginLeft: '1ch' } }, ['Submission notifications'])]),
```

Comment:
How would you feel about "Receive submission notifications"?

Reply:
Changed in https://github.com/DataBiosphere/terra-ui/pull/3398/commits/c00b05c129b94d37f5f2c076a1b132e3a2a823f6.